src/App.css

```diff
@@ -1,38 +1,82 @@
-.App {
+.container {
```

Comment:
because this is a simple app i'm okay with this way of doing styling. ON a more complicated app this pattern will cause a lot of trouble. Because there isn't a pattern to how you choose to size things. There are mixtures of font sizes and units (px, em). Typically good designs will have a theme or a prescribed way to apply styles. For example font sizes may only come in certain values (8px, 12px, 16px, 20px) and padding/margin (0px, 2px, 4px, 8px, 16px) etc